docs/getting-started/#02-first-app.md

```diff
@@ -8,6 +8,8 @@ and just forwards transactions to the application when they need to be validated
 
 In this guide, we show you some examples of how to run an application using Tendermint.
 
+**Note:** It is highly recommended to read the [Using Tendermint Guide](/docs/guides/using-tendermint) prior to working through this tutorial.
```

Comment:
i found it helpful ... it provides context for what one is doing although there's a lot of duplicate stuff between the getting started and guides